src/parser.rs

```diff
@@ -37,6 +37,44 @@ use crate::ParsedDaysOfWeek;
 use crate::PossibleLiterals;
 use crate::PossibleValue;
 
+/// Options to manipulate the parsing manner.
+#[derive(Debug, Copy, Clone)]
+pub struct ParseOptions {
+    /// Whether fallback to UTC when the timezone part is missing.
+    ///
+    /// Default to `false`.
+    pub fallback_to_utc_timezone: bool,
+
+    /// Whether fallback to the system timezone when the timezone part is missing.
+    ///
+    /// Default to `false`.
+    pub fallback_to_system_timezone: bool,
```

Comment:
What happens if both fallbacks are enabled..? OK, you return an error below. But it would be much nicer to ensure they are exclusive in the type system via an enum. As it stands today, someone can build an invalid `ParseOptions` and then it's on them to explicitly call `validate`. That's not a great API IMO.

Reply:
Updated as an enu,.